yogstation/code/modules/scripting/Interpreter/Objects.dm

```diff
@@ -0,0 +1,214 @@
+GLOBAL_LIST_EMPTY(ntsl_methods)
+
+/n_Interpreter/proc/get_property(object, key, scope/scope, node)
+	if(islist(object))
```

Comment:
So are these list objects going to just, like, co-exist with vectors?

Reply:
vectors are lists.

Reply:
So you've removed the whole ``vector()`` business and replaced it with ``list()`` and a system of methods?